samples/ControlGallery/ControlGallery/Views/GesturePlayground.razor

```diff
@@ -0,0 +1,38 @@
+﻿@page "/gestureplayground"
+
+<ContentView>
+    <StackLayout Margin="new Thickness(20)">
+
+        <Label Text="Tap me or pan me"
+               FontSize="40" GestureRecognizers="@GetLabelGestures()" BackgroundColor="Color.LightCoral" />
+
+        <Label Text="@($"The label above was clicked {count} times")" FontSize="@fontSize" BackgroundColor="Color.AliceBlue" />
+
+    </StackLayout>
+</ContentView>
+
+@code
+{
+    int count;
+    double fontSize = 20;
+
+    List<IGestureRecognizer> GetLabelGestures()
+    {
+        return new List<IGestureRecognizer>() {
+            new TapGestureRecognizer{NumberOfTapsRequired=1,  }.AddEvents(g => g.Tapped += OnLabelTapped ),
```

Comment:
I remember when I first try `TapGestureRecognizer`, it didn't work if `NumberOfTapsRequired` was more than 1. I'm curious if it's working now.

Reply:
I just tested this out and the number of taps is ignored and it always requires 1 tap.

Reply:
NumberOfTapsRequired is working correctly. I just pulled down this branch and added a double and triple tap.

Reply:
NumberOfTapsRequired is working correctly, or at least it works consistently with Xamarin Forms. I pulled down the branch and added a double tap.

Android only supports 1 or 2 taps. On iOS you can have more. 

https://stackoverflow.com/questions/47372959/element-triple-tap

Reply:
I only tried with 1. Ultimately this code does whatever Xamarin does, so if there's a bug, I wouldn't be responsible for it 😄